docs/section5/Rule5-46.md

```diff
@@ -0,0 +1,40 @@
+
+# Envelope - Rule 5-46  
+
+**Rule ID:** 5-46  
+**Rule Description:** The infiltration shall be modeled using the same methodology and adjustments for weather and building operation in both the proposed design and the baseline building design.  
+**Rule Assertion:** For all zones, B-RMR infiltration:infiltration_modeling_method_name= P-RMR infiltration:infiltration_modeling_method_name.  
```

Comment:
Let's also add AND infiltration:modeling_method = P-RMR infiltration:modeling_method

Reply:
Updated.